Assets/Scripts/Interaction/Hand.cs

```diff
@@ -129,8 +134,16 @@ void updateSigilAnchor()
 
     void OnTriggerEnter(Collider other) {
         if (IS_PRIMARY) return;
-
+        currentProjectile = other.gameObject;
         if (getSpeed() > speedThreshold) {
+            if (!IS_PRIMARY)
+            {
+                if (other.tag == "Projectile")
+                {
+                    counterProjectile();
+                    GetComponent<Projectile>().reflect();
```

Comment:
you are calling this in the current scope, you should be calling other.GetComponent if the function is inside the projectile. The hand does not have a Projectile Component to retreive, as far as I know.